internal/prompt/prompt.go

```diff
@@ -318,18 +274,31 @@ func Seed(reader *bufio.Reader) (seed []byte, imported bool, err error) {
 				fmt.Printf("Input error: %v\n", err.Error())
```

Comment:
What do you think about using a logger with a level for error?

Reply:
I think this is not advisable in this case. This is interactive tool, not daemon.